layouts/index.html

```diff
@@ -0,0 +1,157 @@
+                            </p>
+                        </div>
+                    </div>
+                </div>
+                <div class="col">
+                    <div class="card w-100">
+                        <img
+                            class="card-img-top img-fluid"
+                            src="https://d33wubrfki0l68.cloudfront.net/e2ad15d073dbd9d0a734261924994fc7fac2f302/bc5f9/files/classroom.jpg"
+                            alt=""
+                            width="348"
+                            height="196"
+                        >
+                        <div class="card-body">
+                            <h5 class="card-title">
+                                <a class="card-link" href="/services/courses/">Courses Database</a>
+                            </h5>
+                            <p class="card-text">We provide a database of student-sourced course reviews and descriptions to help you choose you next classes.</p>
+                        </div>
+                    </div>
+                </div>
+                <div class="col">
+                    <div class="card w-100">
+                        <img
+                            class="card-img-top img-fluid"
+                            src="https://d33wubrfki0l68.cloudfront.net/e71f6e23e6c2a230223c6e13f9eb1ab25244b27e/00822/files/chips.jpg"
+                            alt=""
+                            width="348"
+                            height="196"
+                        >
+                        <div class="card-body">
+                            <h5 class="card-title">
+                                <a class="card-link" href="/cube/menu/">Cube Menu</a>
+                            </h5>
+                            <p class="card-text">We provide cheap snacks, like chips, pizza, candy, and drinks, in the Cube.</p>
+                        </div>
+                    </div>
+                </div>
+            </div>
+            <div class="row justify-content-center">
+                <div class="col-lg-8 text-center large-text pb-5">
+                    <h1>Calendar</h1>
+                </div>
+            </div>
+            <iframe
+                id="open-web-calendar"
+                style="background:url('https://raw.githubusercontent.com/niccokunzmann/open-web-calendar/master/static/img/loaders/circular-loader.gif') center center no-repeat;"
+                src="https://open-web-calendar.herokuapp.com/calendar.html?url=https%3A%2F%2Fubccsss.org%2Fevents%2Findex.ics&amp;title=UBC%20CSSS%20Calendar"
+                sandbox="allow-scripts allow-same-origin allow-top-navigation"
+                allowTransparency="true"
+                scrolling="no"
+                frameborder="0"
+                height="600px"
+                width="100%"
+                class="px-5 pb-5"
+            ></iframe>
+        </div>
+        {{ partial "sidebar" . }}
```

Comment:
Might want to disable this on mobile and tablet as it's a bit confusing to go from services > tags > elsewhere > calendar.